homeassistant/components/websocket_api/http.py

```diff
@@ -95,6 +92,8 @@ def _send_message(self, message):
         Async friendly.
         """
         try:
+            if not isinstance(message, str):
```

Comment:
Shouldn't this take place outside the try block?

Reply:
Yes it should 👍

Reply:
Fixed.. too focused on the problem and not enough on the code here.